File: TransCoder_tokenized_test_set_functions/cpp/SQUARES_OF_MATRIX_DIAGONAL_ELEMENTS_1.cpp
void diagonalsquare ( int mat [ ] [ MAX ], int row, int column ) {
  cout << " \nDiagonal one : ";
  for ( int i = 0;
  i < row;
  i ++ ) {
    cout << mat [ i ] [ i ] * mat [ i ] [ i ] << " ";
  }
  cout << " \n\nDiagonal two : ";
  for ( int i = 0;
  i < row;
  i ++ ) {
    cout << mat [ i ] [ row - i - 1 ] * mat [ i ] [ row - i - 1 ] << " ";
  }
}

File: results/java_codex/SQUARES_OF_MATRIX_DIAGONAL_ELEMENTS_1.java
void diagonalsquare ( int mat [ ] [ ], int row, int column ) {
  cout << " Diagonal one : " ;
  for ( int i = 0;
  i < row;
  i ++ ) {
    cout << mat [ i ] [ i ] * mat [ i ] [ i ] << " " ;
  }
  cout << endl;
  cout << " Diagonal two : " ;
  for ( int i = 0;
  i < row;
  i ++ ) {
    cout << mat [ i ] [ row - i - 1 ] * mat [ i ] [ row - i - 1 ] << " " ;
  }
}